dashboard/assets/components/Common.jsx

```diff
@@ -0,0 +1,29 @@
+// isNullOrUndefined returns true if the given variable is null or undefined.
+export const isNullOrUndefined = variable => variable === null || typeof variable === 'undefined';
+
+// defaultZero returns 0 if the given element is null or undefined, otherwise returns the given element.
+export const defaultZero = elem => isNullOrUndefined(elem) ? 0 : elem;
+
+export const MEMORY_SAMPLE_LIMIT = 200; // Maximum number of memory data samples.
+export const TRAFFIC_SAMPLE_LIMIT = 200; // Maximum number of traffic data samples.
+
+// The sidebar menu and the main content are rendered based on these elements.
+export const TAGS = (() => {
+    const T = {
+        home: { title: "Home", },
+        logs: { title: "Logs", },
+        networking: { title: "Networking", },
+        txpool: { title: "Txpool", },
+        blockchain: { title: "Blockchain", },
+        system: { title: "System", },
```

Comment:
Lets use the following order:

 * Home
 * Chain
 * Transactions
 * Network
 * System
 * Logs